assets/styles/abstracts/_theme.scss

```
@@ -65,6 +66,7 @@ $themes: (
     'k-accentlight2-dark': #ffffff,
     'k-accentlight2-dark-head': #ffffff,
     'k-accentlight2-dark-paragraph': #b9b9b9,
+    'k-accentlight3': "#FFAEE1",
```

```suggestion
    'k-accentlight3': '#ffaee1',
```
since it's the same color between dark & light mode why don't we just create a single var